rust/rsc/src/rsc_tool/main.rs

```diff
@@ -166,6 +170,33 @@ async fn remove_all_jobs(db: &DatabaseConnection) -> Result<(), Box<dyn std::err
     Ok(())
 }
 
+async fn bootstrap_db(db: &DatabaseConnection) -> Result<(), Box<dyn std::error::Error>> {
+    // Get all the info from the user
+    let use_dev_key = Confirm::new("Create a test API key? (development only!)")
+        .with_default(false)
+        .prompt()?;
+    let key_desc = Text::new("What is the key used for?").prompt()?;
+    let local_store_root = Text::new("Where should local blobs be saved?").prompt()?;
+
+    // Create the API key
+    let key = if use_dev_key {
+        Some("InsecureKey".into())
+    } else {
+        None
+    };
+    add_api_key(key, key_desc, db).await?;
+
+    // Create the local blob store
+    add_local_blob_store(local_store_root, db).await?;
+
+    // Create "DBOnly blob store"
+    database::create_dbonly_blob_store(db).await?;
+
+    println!("");
+    println!("Done! Use the store id to set the active store and share the key as appropiate.");
```

Comment:
Setting the active store is something that user does via another rsc_tool invocation?

Reply:
Its a part of the launch parameters into the RSC. You aren't allowed to change the active store while the RSC is running since the "activation" of a store is a non trivial action on a nearly global hashmap

Reply:
Should probably add a README to `rsc_tool` to cover common use patterns that aren't covered by `--help`